### ISSUE TITLE
When I paste some words from Bing directory, the VSCode will crash.

### ISSUE THREAD

**Issue by Li-Mark** [Microsoft/vscode#26452]
- VSCode Version: Code 1.12.1 (f6868fce3eeb16663840eb82123369dec6077a9b, 2017-05-04T21:26:50.689Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|beautify|HookyQR|1.0.2|;

---

Steps to Reproduce:

1. Open Bing directory, (UWP, version 2.7.1.0)
2. Search a word.
3. Copy the word and paste the word in VSCode
4. Both of the bing directory and vscode will crash

**Comment by egamma** [comment]
@Li-Mark can you please explain what you mean by 'Open Bing directory', how can we reproduce it? 
Can you reproduce the crash in other ways that do not require the Bing directory?

**Comment by Li-Mark** [comment]
@egamma I translated some words with Microsoft Bing Dictionary (UWP, Chinese to English) . When I copied the result to vscode, the dictionary and vscode crashed. I got some error events from the windows event viewer.
---------------------------------------
The program Code.exe version 1.12.1.0 stopped interacting with Windows and was closed. To see if more information about the problem is available, check the problem history in the Security and Maintenance control panel.
 Process ID: 2990
 Start Time: 01d2ca4e80ad2d67
 Termination Time: 17
 Application Path: C:\Program Files (x86)\Microsoft VS Code\Code.exe
 Report Id: ea4da29c-3641-11e7-9343-28d244bdec32
 Faulting package full name: 
 Faulting package-relative application ID: 
-------------------------------------------------------
Package MSNChina.Win10_2.7.1.0_x64__ybyqcqtwzm05j+App was terminated because it took too long to suspend.
---------------------------------------

But two hours later, the issue cannot be reproduced. I'm sorry to have troubled you.

**Comment by Li-Mark** [comment]
@egamma I translated some words with Microsoft Bing Dictionary (UWP, Chinese to English) . When I copied the result to vscode, the dictionary and vscode crashed. But two hours later, the issue cannot be reproduced. I'm sorry to have troubled you.

I got some error events from the windows event viewer after the vscode crashed.
***
Event 2484, Apps:  5/11/2017 8:02:41 PM

Package MSNChina.Win10_2.7.1.0_x64__ybyqcqtwzm05j+App was terminated because it took too long to suspend.
***
Event 1001, Windows Error Reporting: 5/11/2017 8:02:42 PM

Fault bucket , type 0
Event Name: AppHangB1
Response: 不可用(unavailable)
Cab Id: 0

Problem signature:
P1: Code.exe
P2: 1.12.1.0
P3: 58e7ba2a
P4: e5ba
P5: 134217728
P6: 
P7: 
P8: 
P9: 
P10: 

Attached files:
\\?\C:\Users\liyun\AppData\Local\Temp\WER701B.tmp.appcompat.txt
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER704B.tmp.WERInternalMetadata.xml

These files may be available here:
C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppHang_Code.exe_795ab59e9e1e3768bf9612f7b4d930aea3417e8_6710a98d_33df74ce

Analysis symbol: 
Rechecking for solution: 0
Report Id: ea4da29c-3641-11e7-9343-28d244bdec32
Report Status: 97
Hashed bucket: 
***
Event 1002, Application Hang: 5/11/2017 8:02:42 PM

The program Code.exe version 1.12.1.0 stopped interacting with Windows and was closed. To see if more information about the problem is available, check the problem history in the Security and Maintenance control panel.
Process ID: 2990
Start Time: 01d2ca4e80ad2d67
Termination Time: 17
Application Path: C:\Program Files (x86)\Microsoft VS Code\Code.exe
Report Id: ea4da29c-3641-11e7-9343-28d244bdec32
Faulting package full name:
Faulting package-relative application ID:
***
